app/models/versioning/revision.rb

```diff
@@ -11,6 +11,12 @@ class Revision < ApplicationRecord
                foreign_key: :created_by_id
     # rubocop:enable Rails/InverseOf
 
+    has_many :current_for_editions,
```

Comment:
This commit suggests that a revision could have multiple editions, though I'm not sure I understand how that could happen: https://github.com/alphagov/content-publisher/pull/672/commits/0d03980c69c48705f7fd12b22c852422b89da3ff

Reply:
@emmabeynon since an edition points to a particular revision there isn't really any reason why different editions can't point to the same revision - particularly if they have the same content. For instance when you create a new edition it could point to the same revision and if we were to support the ability to roll a document back to a particular point in time.

In practicality it is unlikely that a revision will be shared with multiple editions since we reset the change note at the point of creating a new edition

Reply:
Will be removed by disabling inverse_of cop :-).